modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -89,8 +106,8 @@ Creating a vector index requires link:{neo4j-docs-base-uri}/operations-manual/{p
 The index name must be unique among both indexes and constraints.
 ====
 
-A vector index is a single-label, single-property index for nodes.
-A vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `2048` _inclusive_), and the measure of similarity between two vectors (case-insensitive `STRING`).
+A vector index is a single-label, single-property index for nodes or a single-relationship-type, single-property index for relationships.{fn-provider-differences}
+A vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `4096` _inclusive_),{fn-provider-differences} and the measure of similarity between two vectors (case-insensitive `STRING`).
```

Comment:
Two references to the same footnote, without another footnote in between, in the same paragraph might not be necessary. 

```suggestion
A vector index is a single-label, single-property index for nodes or a single-relationship-type, single-property index for relationships.
A vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `4096` _inclusive_), and the measure of similarity between two vectors (case-insensitive `STRING`).{fn-provider-differences}
```